### ISSUE TITLE
Forum Upduck button lacks tooltip

### ISSUE THREAD

**Issue by williamjallen** [Submitty/Submitty#10364]
The new "upduck" button introduced in https://github.com/Submitty/Submitty/pull/10017 does not have a tooltip indicating what the button does.  Given that it is a new feature, having such a tooltip may help reduce confusion about the purpose of the button.
